run.sh

```diff
@@ -11,4 +11,4 @@ fi
 # Load .env file
 export $(egrep -v '^#' $DIR/.env | xargs)
 
-python3 ./battleship.py
\ No newline at end of file
+python ./battleship.py
```

Comment:
What is the purpose of this change? The application was written/tested in python3.9. Arguably that one should be selected for in case a user has both 2 and 3 installed on their system. A lot of macs do this because of xcode/swift stuff